src/openrct2/ride/RideRatings.cpp

```diff
@@ -4440,16 +4440,16 @@ static const ride_ratings_calculation ride_ratings_calculate_func_table[RIDE_TYP
     ride_ratings_calculate_magic_carpet,                   // MAGIC_CARPET
     ride_ratings_calculate_submarine_ride,                 // SUBMARINE_RIDE
     ride_ratings_calculate_river_rafts,                    // RIVER_RAFTS
-    NULL,                                                  // 50
+    nullptr,                                                  // 50
```

Comment:
the comments here are now misaligned

Reply:
The new run-clang-format job did mark this too!